pnpm-lock.yaml

```diff
@@ -20,8 +20,13 @@ importers:
       pnpm: 4.6.0
       prettier: 1.19.1
       prettier-plugin-package: 0.3.1_prettier@1.19.1
-      rollup: 1.27.14
+<<<<<<< HEAD
```

Comment:
Merge conflicts?

Reply:
Yes something went wrong syncing, fixed it now